include/envoy/grpc/BUILD

```diff
@@ -17,3 +17,8 @@ envoy_cc_library(
         "//include/envoy/http:header_map_interface",
     ],
 )
+
+envoy_cc_library(
+    name = "status_interface",
```

Comment:
Nit: Could just call this `"status"`, since it's not as much an interface as an independent interface-level library (we do something similar for `optional`).